src/Uno.UI/UI/Xaml/Media/AcrylicBrush/Android/AcrylicBrush.Android.cs

```diff
@@ -0,0 +1,42 @@
+﻿using System;
+using Android.Graphics;
+using Android.Graphics.Drawables;
+using Android.Views;
+using Uno.Disposables;
+using Uno.UI.Controls;
+
+namespace Windows.UI.Xaml.Media
+{
+	public partial class AcrylicBrush
+    {
+		bool ready = false;
+        internal IDisposable Apply(BindableView owner)
+		{
+			View v = owner;
```

Comment:
Codacy found an issue: [Remove the unused local variable 'v'.](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=5932087)